game.cpp

```diff
@@ -1430,7 +1450,7 @@ void test_game() //!OCLINT tests may be many
   }
 #endif
 
-  //#define FIX_ISSUE_241
+  #define FIX_ISSUE_241
```

Comment:
This test passes, so the `#define`s and other useless fluff can be removed. I'll make that into a newbie Issue :-)